packages/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -6,7 +6,11 @@ const userNoteInput = () => $('[data-test-id="user-note"] textarea');
 
 const allergenLink = () => $('[data-test-id="allergy-button"]');
 
-const fulfillmentTimeDropdown = () => $('[data-test-id="fulfillment-time"]');
+// Dropdown
+
+const dropdownSelector = () => $('[data-test-id="form-select"]');
+
+const dropdownOptions = () => $$("select option");
```

Comment:
Is it worth making this selector more specific, in case there are ever more dropdowns added?

Reply:
Yeah I think that makes sense, it's very general the moment - will change it to "time-dropdown" to be more specific 👍

Reply:
Just to clarify I mean the CSS selector (although you can update the variable name too if you like)
```suggestion
const orderTimeDropdownOptions = () => $$('[data-test-id="form-select"] select option');
```
Would this work?

Reply:
Ah okay I see, yep happy to change this - much clearer